ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -963,6 +963,16 @@ def uninstall_master(host, ignore_topology_disconnect=True,
 
     result = host.run_command(uninstall_cmd)
     assert "Traceback" not in result.stdout_text
+
+    # Check that IPA certs have been deleted after uninstall
+    # Related: https://pagure.io/freeipa/issue/8614
+    assert host.run_command(['test', '-f', paths.IPA_CA_CRT],
+                            raiseonerr=False).returncode == 1
+    assert host.run_command(['test', '-f', paths.IPA_P11_KIT],
+                            raiseonerr=False).returncode == 1
+    assert host.run_command("trust list | grep IPA",
```

Comment:
You can use shlex for this. You can grep in ipatests/ to see how it works.

Reply:
Or you could fetch the whole output (e.g. not use grep) and use substring matching instead.